From 37a3f1c43077ea5abac74b712921f6a2cd663f39 Mon Sep 17 00:00:00 2001
From: Steve Hay <me@stevenhay.com>
Date: Fri, 31 Mar 2023 09:02:20 +0200
Subject: [PATCH] Configure.ac / Remove pthreads check.

This is a hack, but need to adjust the phtreads check to be arm aware.

Signed-off-by: Steve Hay <me@stevenhay.com>
---
 configure.ac | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/configure.ac b/configure.ac
index a62e8df..d5d5a76 100644
--- a/configure.ac
+++ b/configure.ac
@@ -110,18 +110,6 @@ AS_IF([test "$enable_sdl" != "no"], [
 AC_ARG_ENABLE([threading],
 AS_HELP_STRING([--enable-threading], [multhreading]),
 [], [enable_threading=yes])
-AS_IF([test "x$enable_threading" = "xyes" && ! test "$EMSCRIPTEN"], [
-  m4_include([m4/autoconf-archive/ax_pthread.m4])
-  AX_PTHREAD([
-    AC_DEFINE([USE_THREADS], [1], [Define USE_THREADS])
-    LIBS="$LIBS $PTHREAD_LIBS $PTHREAD_CFLAGS"
-    CFLAGS="$CFLAGS $PTHREAD_CFLAGS"
-    CXXFLAGS="$CXXFLAGS $PTHREAD_CFLAGS"
-    echo "LIBS=$LIBS"
-  ], [
-    AC_MSG_ERROR([pthreads not found])
-  ])
-])
 
 AC_ARG_ENABLE([gles],
   AS_HELP_STRING([--enable-gles], [OpenGL ES support]),
-- 
2.30.2

